dom0/sys-usb.top

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+base:
+  dom0:
```

Comment:
I think it shouldn't be dom0 here, but rather sys-usb. I think this will resolve the issue i've experienced where it copies the files to dom0

Reply:
Yep. Major testing fail when I tried to get the `sys-usb` added to the entire Salt run. Fixed.